app/Main.hs

```diff
@@ -27,15 +27,19 @@ githubKeyToParse = optional (
       <> Opt.metavar "DEVELOPER_KEY"
       <> Opt.help "A github developer key to allow for more API calls for the IssueTracker checker"))
 
+noColorParse :: Opt.Parser Bool
+noColorParse = not <$> Opt.switch (
+  Opt.long "no-colors"
+    <> Opt.help "Disable colored outputs")
+
 optionsParser :: Opt.Parser KrankOpts
 optionsParser = KrankOpts
   <$> filesToParse
   <*> (KrankConfig
        <$> githubKeyToParse
        <*> (Opt.switch $ Opt.long "dry-run"
         <> Opt.help "Perform a dry run. Parse file, but do not execute HTTP requests")
-      <*> (not <$> (Opt.switch $ Opt.long "no-colors"
-           <> Opt.help "Disable colored outputs"))
+      <*> noColorParse
```

Comment:
nit: indentation seems still inconsistent

Reply:
You are right. I think we'll soon use a tool such as `ormolu` to format for us.